federatedml/param/param.py

```diff
@@ -150,7 +150,7 @@ def __init__(self, method=None, train_input_table=None, train_input_namespace=No
 
 
 class InitParam(object):
-    def __init__(self, init_method='random_uniform', init_const=1, fit_intercept=True):
+    def __init__(self, init_method='random_uniform', init_const=1, fit_intercept=False):
```

Comment:
why change fit_intercept default value to False?

Reply:
It‘s’ a mistake of me, I will fix it right now. The vesion of six development machines is too old, maybe we should update them with the newest version in github